class.jetpack-jitm.php

```diff
@@ -414,6 +426,62 @@ function videopress_media_upload_warning_msg() {
 		<?php
 	}
 
+	/**
+	 * Display message prompting user to install WooCommerce Services.
+	 *
+	 * @since 4.6
+	 */
+	function woocommerce_services_msg() {
+		if ( ! current_user_can( 'manage_woocommerce' ) ) {
+			return;
+		}
+
+		if ( isset( self::$jetpack_hide_jitm['woocommerce_services'] ) ) {
+			return;
+		}
+
+		if ( ! function_exists( 'wc_get_base_location' ) ) {
+			return;
+		}
+
+		$base_location   = wc_get_base_location();
+		$store_in_usa    = ( 'US' == $base_location['country'] );
+		$store_in_canada = ( 'CA' == $base_location['country'] );
+
+		if ( $store_in_usa ) {
```

Comment:
I'm not sure I see the point of a throwaway variable when the `'US' === $base_location['country']` could be tested inline here?